x/asset/client/cli/tx.go

```diff
@@ -810,3 +810,127 @@ func NewCreateAssetInAppMsg(clientCtx client.Context, txf tx.Factory, fs *flag.F
 	}
 	return txf, msg, nil
 }
+
+func NewCmdSubmitAddMultipleAssetsPairsProposal() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "add-multiple-assets-pairs [flags]",
+		Args:  cobra.ExactArgs(0),
+		Short: "Submit multiple assets and pairs",
+		Long: `Must provide path to a add assets in JSON file (--add-assets) describing the asset in app to be created
```

Comment:
double check this, update the comment according to txs.

Reply:
Updated